tests/firefox/tabbed_browser/stop_sending_referer_details_on_text_links.py

```diff
@@ -0,0 +1,67 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from targets.firefox.fx_testcase import *
+
+
+class Test(FirefoxTest):
+    @pytest.mark.details(
+        description="Stop sending referer details on text links",
+        locale=["en-US"],
+        test_case_id="139614",
+        test_suite_id="2103",
+    )
+    def run(self, firefox):
+        raw_paste_data_pattern = Pattern("raw_paste_data.png")
+        what_is_my_referer_link_pattern = Pattern("whatismyreferer_link.png")
+        what_is_my_referer_link_selected_pattern = Pattern("whatismyreferer_link_selected.png")
+        your_http_referer_pattern = Pattern("your_http_referer.png")
+        no_referer_hidden_pattern = Pattern("no_referer_hidden.png")
+
+        navigate("https://pastebin.com/QD6QHKn3")
```

Comment:
This site can’t be reached

Reply:
Please explain a little more about the issue as I can navigate to this site. 

![image](https://user-images.githubusercontent.com/54833244/71783189-27469f00-3009-11ea-9f8c-2eec8fcb8e40.png)